packages/odyssey-react/src/components/Checkbox/Checkbox.module.scss

```diff
@@ -46,58 +46,58 @@
   &:indeterminate {
     + .label {
       .indicator {
-        color: var(--ColorCheck);
+        color: var(--CheckColor);
```

Comment:
Are we not using `TextColor` here for this CSS style declaration here? Seems like it should follow that pattern but on the other hand I can see folks thinking the check or really any indicator are not text...